### DISCUSSION TITLE
Release 0.9.9

### DISCUSSION THREAD

**Discussion by jon-harper** [jon-harper/OmniBox#73] [Announcements]
## Major New Features

- Universal PSU mount
- Second CPU bay added
- CPU bays now in rear
- New bottom panel and side panels
- 20+ new zip tie anchors
- Lots of fixes and support for new products.
- New assembly documentation
- Assembly videos
- Documentation is now release-specific (versioned)

See the [release log](https://jon-harper.github.io/OmniBox/0.9.9/history/) for a full changelog.

## Existing 0.9.8 Users

If you are already building an 0.9.8 case, the `main` branch is snapshotted and stored in the `0.9.8-frozen` branch.

**Comment by jon-harper** [comment]
Full disclosure: there is no stackable lid replacement for the moment. I'm using a carry handle lid without the handle (and it's doing fine, thus the lack of rush).

**Comment by TheoBromine2022** [comment]
Nice. I just spent the past 2 days tweaking marlin to get the e5+ and the octopus to work together, all that is left is to get the extruder working right. I think I can finally get this thing running on the barebones .9.8 box.

I am so stoked to get this thing operational.

**Reply by jon-harper** [reply]
That's awesome news! I know you've wanted that E5+ back up. Cheers!

**Reply by TheoBromine2022** [reply]
yeah. cross your fingers because once this firmware works I can slap another octopus onto another e5+ and get this farm running. god what a pain in the ass, I kept giving up because vscode and platformio kept fucking with me, but autobuildmarlin downloaded something yesterday that platformio didnt and all of a sudden I had a working complier. 

That being said, is there a tray mount for that BTT capacitor setup for power failure? I noticed it was on black friday sales yesterday but didnt buy it. I probably should have looked at APC UPS systems for black friday pricings now that I think about it...

**Reply by jon-harper** [reply]
The BTT UPS 24V does have a tray. Do I trust it a PCB covered in capacitors? *cough*

That said, you should be prepared: a battery backup that will handle the draw of the E5+'s bed heating is on the high end. An APC 750VA can't handle the current draw of an oversized AC bed heating and will trip alarms. You'll want to poke around the specs to see what's available that can handle that much current on one outlet.
